constants/InjiConstants.js

```diff
@@ -0,0 +1,14 @@
+//Banner Type Constants
```

Comment:
Instead of introducing new constant file, we should use existing one https://github.com/mosip/inji/blob/main/shared/constants.ts

Reply:
Addressed